lib/stream.h

```diff
@@ -123,6 +123,9 @@ struct stream_fifo {
 #define STREAM_CONCAT_REMAIN(S1, S2, size) ((size) - (S1)->endp - (S2)->endp)
 
 /* deprecated macros - do not use in new code */
+#if CONFDATE > 20181128
+CPP_NOTICE("lib: time to remove deprecated stream.h macros")
+#endif
```

Comment:
What macros are being deprecated? I can see that there are no more occurrences of `STREAM_PNT` and `STREAM_REMAIN`, but `STREAM_DATA` is still being used everywhere.

Reply:
technically all of these have been deprecated since 2005. `STREAM_PNT` and `STREAM_REMAIN` will be removed; `STREAM_DATA` will remain with the current comment discouraging its use but unfortunately we cannot drop that one at this time as you said. I will update this patch to note which ones are slated for removal.